src/push/index.ts

```diff
@@ -70,11 +70,23 @@ export async function push(monitors: Monitor[], options: PushOptions) {
       const { error, message } = await body.json();
       throw formatAPIError(statusCode, error, message);
     }
-    const { failedMonitors } = await body.json();
-    if (failedMonitors.length > 0) {
-      throw formatFailedMonitors(failedMonitors);
+    body.setEncoding('utf-8');
+    for await (const data of body) {
+      // Its kind of hacky for now where Kibana streams the response by
+      // writing the data as NDJSON events (data can be interleaved), we
+      // distinguish the final data by checking if the event was a progress vs complete event
+      const chunks = safeNDJSONParse(data);
+      for (const chunk of chunks) {
+        if (typeof chunk === 'string') {
+          progress(chunk);
+          continue;
+        }
+        const { failedMonitors } = chunk;
+        if (failedMonitors && failedMonitors.length > 0) {
+          throw formatFailedMonitors(failedMonitors);
+        }
+      }
     }
-    done('Pushed');
```

Comment:
@justinkambic Seems like your last merge removed this line.

Reply:
oh weird, sorry it was a mistake merging. Will fix.